app/src/main/java/org/mozilla/fenix/tabstray/browser/BrowserTabsAdapter.kt

```diff
@@ -42,21 +41,17 @@ class BrowserTabsAdapter(
 
     override fun getItemViewType(position: Int): Int {
         return if (context.settings().gridTabView) {
```

Comment:
We can remove this line here and use the `layoutManager` passed in the from the constructor.

We can get some hints on how to setup the `GridLayoutManager` from [here](https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt#L404-L431) and use a `spanCount` of 1 if it's a linear layout.

Reply:
Agreed. I'll fix this in a follow up.